scripts/check_imports.swift

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/swift
```

Comment:
Writing this kind of check in Swift means these checks can't run on Windows. While most of Firebase doesn't support Windows from this repo, Firestore does. Please reconsider this.

Reply:
Swift looks to be coming soon to Windows - next week perhaps?  https://swift.org/blog/5-3-release-process/.